cmd/release-controller/http.go

```diff
@@ -196,7 +196,7 @@ const releaseDashboardPageHtml = `
 							{{ tableLink $release.Config . }}
```

Comment:
You should probably remove all changes in this file - looks like you have old / improperly merged changes (looks like it's reverting master)